docs/FAQ/General-FAQ.md

```diff
@@ -1,5 +1,20 @@
 # General FAQ
 
+## How and what commit is chosen when running `evergreen patch`?
+
+The Evergreen CLI grabs the user's config file and determines what project should be used. The CLI then establishes a connection with the Evergreen server and retrieves that project's information, specifically the GitHub org + repo + branch (which we'll call, org, repo, and main respectively).
+
+The CLI then fetches the corresponding [remote](https://git-scm.com/docs/git-remote) (which we'll call upstream).
+Using that, it runs `git merge-base upstream/main HEAD` to find the common ancestor of the Evergreen tracking branch and the currently checked out branch. This is the commit that will be used as the base for the patch. Changes (including commits and uncommited changes) made after this common ancestor will be included as diff changes in the patch rather than as commits.
+
+Some Caveats to be aware of:
+
+- Projects can configure an oldest allowed merge-base in their project settings
```

Comment:
Should we add a note on what will happen if the `git merge-base upstream/main HEAD` is not allowed?

Reply:
Ty! This was an incomplete sentence, my bad 😓